### PR TITLE
Add contact form to portfolio project

### PR DESCRIPTION
**Pull request by Talha-Hanif5666**
In the pull request, I have added the contact form with the pair programming technique. I have used HTML and CSS for this.
The portfolio project contains a mobile responsive website page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -189,10 +189,46 @@ <h2 class="about-primary-text">About me</h2>
...
+  <div class="bottom">
+<p class="text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+<form action="https://formspree.io/f/mlevodye" method="POST" >
+<input type="text" id="name" name="user_name" maxlength="30" placeholder="Full name" required>
+<input type="email" id="mail" name="user_email" placeholder="Email address" required>
+<textarea type="text" name="message" placeholder="Enter text here" maxlength="500" required></textarea>
+<button type="submit" value="submit">Get in touch</button>
+</form>
+  </div>
+</section>
```
Comment (Nemwel-Boniface) [inline index.html:212]: - [ ] Good job styling your portfolio contact form section. However, according to the project requirements, you are required to use the CSS flexbox to align the items of this section. Kindly do so.

image

File: index.html [hunk 2]
```diff
@@ -189,10 +189,46 @@ <h2 class="about-primary-text">About me</h2>
...
+      <li><a href="#" id="twitter">
+          <img src="Images\bottom4.png" alt="twitter"></a></li>
+      <li><a href="#" id="medium">
+          <img src="Images\bottom5.png" alt="medium"></a></li>
+    </ul>
+   </div>
+     <div class="home">
+        <img src="Images\Shape.png" alt="home.png">
+     </div>
+</footer>
```
Comment (Nemwel-Boniface) [inline index.html]: - [ ] There seems to be an extending white space after the footer section on your webpage which is not supposed to be there. Kindly fix this.

**This is the white space I am talking about:**

[Screencast from 01-11-2022 04:10:28 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/199240904-f2e2de2c-c817-47c9-a81b-7cde3d4144aa.webm)

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @Talha-Hanif5666 , @stevenmukama  and @codehass    :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Great job working on the portfolio so far! :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -189,10 +189,46 @@ <h2 class="about-primary-text">About me</h2>
...
+<form action="https://formspree.io/f/mlevodye" method="POST" >
+<input type="text" id="name" name="user_name" maxlength="30" placeholder="Full name" required>
+<input type="email" id="mail" name="user_email" placeholder="Email address" required>
+<textarea type="text" name="message" placeholder="Enter text here" maxlength="500" required></textarea>
+<button type="submit" value="submit">Get in touch</button>
+</form>
+  </div>
+</section>
+
+<footer>
```
Comment (victorokoroji) [inline index.html:214]: - [ ] As the previous reviewer suggested, there seems to be an extending white space after the footer section on your webpage which is not supposed to be there. Kindly fix this.

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @Talha-Hanif5666, 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by fobadara — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -564,18 +564,42 @@
...
 
-a:hover,
-ul li:hover {
+a:hover , ul li:hover {
   background-color: #ff6b00;
+  cursor: pointer;
+}
+
+
+.content-defaults a:hover {
```
Comment (leehaney254) [inline style.css:575]: Kindly use a single space to separate your blocks. :smile:

**Review by Lucas-Erkana — COMMENTED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @Talha-Hanif5666 
# STATUS: APPROVED :green_circle: :clap: 

Your project is complete!  There is nothing else to say other than... 
Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Great job Implementing the changes provided by the previous reviewers ✅
- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- No Linter errors ✅


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

- [OPTIONAL] I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Contact form for Added ✅
- Formspree works ✅
- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- No Linter errors ✅



HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

- [OPTIONAL] I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leehaney254 — COMMENTED** [review]
Some good work. :muscle:

**Comment by Rana-Mudassir** [comment]
Good Work. Everything is working well. But it would be better if you use comments after sections/feature so it becomes more readable to other collaborators.
Happy Coding 👍

**Pull request merged**